server/src/metadata/field-metadata/field-metadata.entity.ts

```diff
@@ -10,28 +10,33 @@ import {
   UpdateDateColumn,
 } from 'typeorm';
 
-import { FieldMetadataInterface } from 'src/workspace/workspace-schema-builder/interfaces/field-metadata.interface';
+import { FieldMetadataInterface } from 'src/metadata/field-metadata/interfaces/field-metadata.interface';
 import { FieldMetadataTargetColumnMap } from 'src/metadata/field-metadata/interfaces/field-metadata-target-column-map.interface';
 import { FieldMetadataDefaultValue } from 'src/metadata/field-metadata/interfaces/field-metadata-default-value.interface';
+import { FieldMetadataOptions } from 'src/metadata/field-metadata/interfaces/field-metadata-options.interface';
 
 import { ObjectMetadataEntity } from 'src/metadata/object-metadata/object-metadata.entity';
 import { RelationMetadataEntity } from 'src/metadata/relation-metadata/relation-metadata.entity';
 
 export enum FieldMetadataType {
   UUID = 'UUID',
   TEXT = 'TEXT',
+  // Temporary workaround for composite field with wrong type
+  VARCHAR = 'VARCHAR',
   PHONE = 'PHONE',
```

Comment:
can we also remove phone, email, and probability that are not supported yet or will never be (for probability)